### DISCUSSION TITLE
Trouble migrating 1.0 develop branch

### DISCUSSION THREAD

**Discussion by davewiththenicehat** [evennia/evennia#2442] [Evennia Questions] [answered]
In `master` branch.
I can run an instance of evennia with my code.  On the same server I can run an instance of evennia `mygame` server. Where the code is an unchanged instance of evennia.
I use an apache virtual host to forward to the new instance.

To put it in an example of directory:
`/home/dave/muddev/daveworld`, my custom evennia instance
`/home/dave/muddev/mygame`, vanillia evennia code.

In the master branch this all works with no issues.

When I attempt to do this in the `develop` branch I receive migration messages on migration.
Example:
![image](https://user-images.githubusercontent.com/54369722/122643737-84d88980-d0df-11eb-8bce-71557c9c4da4.png)

It's obviously attempting to import modules from the code I have made, when that code does not exist in the basic game code.

Any tips on how to run two instances of a game within the `develop` branch would be greatly appreciated. As I contrib to evennia I would like to test my changes in a vanilla instance.
To answer the first question. I am out of cores on my server. I am already running more VM cores than real cores on my server. So creating yet another server is not an option.

Thanks for taking the time to read this.

**Comment by Griatch** [comment, marked as answer]
I doubt this has to do with the `develop` vs `master` branch. There is no `evennia/objects/migrations/0012_*.py` in neither `master`, nor `develop` branches. My guess is that you have ran `evennia makemigrations` on the entire system at one point, which tries to make migrations of your typeclasses into 'real' database tables - this is not correct and why we are very specific about only running `makemigrations` on your custom model/app when you add it. Delete the 0012_* migration and check so you don't have any other migration created accidentally like this. 

Generally, as long as you don't have clashing ports (which you are solving with your apache proxy it sounds like), migrations within a game dir cannot clash and you can run any number of instances.

**Reply by davewiththenicehat** [reply]
Thank you that was 100% what I had done. Thank You
